tests/dispenser/TestFundCommand.test_fund_command_not_authenticated.approved.txt

```diff
@@ -0,0 +1 @@
+ERROR: Please login first
```

Comment:
We should probably include instruction for how they can e.g. " by executing `algokit dispenser login`"